src/rard/templates/research/anonymousfragment_detail.html

```diff
@@ -41,6 +41,13 @@
                     >{% trans 'Make non-anonymous' %}</button>
                 </div>
             </form>
+            <form novalidate style='margin-left:auto' class='form-inline' action='{% url "anonymous_fragment:duplicate" object.pk %}' method='POST'>
+                {% csrf_token %}
+                <div class='form-group'>
+                    <button type='submit' class='btn btn-link text-info p-0 ml-2'
+                        data-what='fragment'>{% trans 'Duplicate' %}</button>
```

Comment:
Just for consistency with above
```suggestion
                        data-what='anonymous fragment'>{% trans 'Duplicate' %}</button>
```